### PR TITLE
Disable debug build configuration

### PR DESCRIPTION
**Pull request by duclong201**
- Save bitrise workflow to master
- Disable debug mode when build

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+    - set-xcode-build-number@1:
+        is_always_run: true
+        inputs:
+        - build_short_version_string: "$FLUTTER_VERSION"
+        - plist_path: "$IOS_INFO_PLIST"
+    - flutter-build@0:
+        is_always_run: true
+        inputs:
+        - cache_level: none
+        - android_output_pattern: "*build/app/outputs/apk/*/*.apk"
```
Comment (duclong201) [inline bitrise.yml:277]: I have disabled debug build in here

File: bitrise.yml [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+    - set-xcode-build-number@1:
+        is_always_run: true
+        inputs:
+        - build_short_version_string: "$FLUTTER_PLUGIN_VERSION"
+        - plist_path: "$IOS_INFO_PLIST"
+    - flutter-build@0:
+        is_always_run: true
+        inputs:
+        - cache_level: none
+        - android_output_pattern: "*build/app/outputs/apk/*/*.apk"
```
Comment (duclong201) [inline bitrise.yml:129]: Debug mode has been removed.

File: bitrise.yml (hunk 1 above)
Comment (duclong201) [inline bitrise.yml:277]: Also here

**Comment by duclong201** [comment]
When the PR is approved and merged, I will update Bitrise to use the workflow stored here.

**Review by datkinnguyen — APPROVED** [review]

**Pull request merged**